### ISSUE TITLE
Cannot use Jitsi Meet with PureBrowser (user agent issue?)

### ISSUE THREAD

**Issue by kwill** [jitsi/jitsi-meet#2147]
**Steps to reproduce:**

* Open PureBrowser (46.2.0esr+really45.4.0esr+pureos-1pureos2)
* Visit https://meet.jit.si (or a room, like https://meet.jit.si/MacabreVegetablesDiscussRapidly )

**What should happen:**

* Page loads
* Prompted for WebRTC access (microphone and camera)
* Able to take part in a meeting

**What happens instead:**

* Page does not finish loading (grey background, favicon visible, nothing else)
* JavaScript error in console (looks like it's related to user agent, see below)
* No WebRTC prompts

**Notes:**

* PureBrowser is a fork of Firefox for Pure OS. It has a user agent of "Mozilla/5.0 (X11; Linux x86_64; rv:45.0) Gecko/20100101 PureBrowser/46.2.0"

* Related report here: https://tracker.pureos.net/T201

* Confirmed that media.peerconnection.enabled (under about:config) is True

* Successfully prompted for WebRTC access (microphone and camera) when visiting https://test.webrtc.org/

* Console error message (when visiting https://meet.jit.si ):

```
(TIME) index.html loaded:	 2608.175 meet.jit.si:9:9
TypeError: navigator.userAgent.match(...) is null
 lib-jitsi-meet.min.js:2:1818
DEPRECATED JS - Cookie has been deprecated since 5.8.0 and will be removed in a future release. Use cookie instead.   
 t.default@https://cdn.jitsi.net/2250/libs/app.bundle.min.js?v=2250:2:3822 app.bundle.min.js:2:116542
mutating the [[Prototype]] of an object will cause your code to run very slowly; instead create the object with the correct initial [[Prototype]] value using Object.create app.bundle.min.js:2:1784342
ReferenceError: JitsiMeetJS is not defined
 app.bundle.min.js:2:370578
```

**Comment by saghul** [comment]
Yep, looks like an extra check would be needed in order to treat PureBrowser as Firefox. The logic will probably need to go here: https://github.com/jitsi/lib-jitsi-meet/blob/master/modules/RTC/RTCBrowserType.js if you're up for witing a PR :-)

**Comment by pejakm** [comment]
I was able to enter a room by changing `general.useragent.compatMode.firefox` to _**true**_. But jitsi could not get my mic and webcam, there's an error:

```
There was an error connecting to your microphone/camera.
i.getSettings is not a function
```

**Comment by saghul** [comment]
Can you paste the full backtrace?

**Comment by pejakm** [comment]
Original browser version is 46.0, I have used `general.useragent.override` to change the version to 56, but it didn't help at all.

Here's the output from js console (original chat messages removed):

```
(TIME) index.html loaded:	 522.865 probnasoba:9:9
(TIME) Sending external connect XHR:	 545.645 external_connect.js:65:5
[modules/RTC/RTCBrowserType.js] <o>:  This appears to be Firefox, ver: 56 lib-jitsi-meet.min.js:2:299646
(TIME) external connect XHR done:	 660.27 external_connect.js:40:13
DEPRECATED JS - Cookie has been deprecated since 5.8.0 and will be removed in a future release. Use cookie instead.   
 t.default@https://cdn.jitsi.net/2441/libs/app.bundle.min.js?v=2441:2:60996 app.bundle.min.js:2:221830
[react/index.web.js] <>:  (TIME) document ready:	 1227.21 app.bundle.min.js:2:725010
[react/features/base/media/middleware.js] <t>:  Start muted:  app.bundle.min.js:2:725010
[react/features/base/media/middleware.js] <t>:  Start audio only set to false app.bundle.min.js:2:725010
[modules/URL/ConferenceUrl.js] <e>:  "Stored original conference URL: https://meet.jit.si/probnasoba" app.bundle.min.js:2:725010
[modules/URL/ConferenceUrl.js] <e>:  "Conference URL for invites: https://meet.jit.si/probnasoba" app.bundle.min.js:2:725010
[modules/RTC/RTCUtils.js] <me</<.value>:  Using the new gUM flow lib-jitsi-meet.min.js:2:299646
[modules/xmpp/strophe.jingle.js] <t>:  Configured STUN servers:  Array [ Object, Object, Object ] lib-jitsi-meet.min.js:2:299646
[modules/xmpp/xmpp.js] <w</<.value>:  (TIME) Strophe Attaching	:1469.72 lib-jitsi-meet.min.js:2:299646
[modules/xmpp/xmpp.js] <w</<.value>:  (TIME) Strophe attached:	 1470.595 lib-jitsi-meet.min.js:2:299646
[modules/xmpp/xmpp.js] <w</<.value>:  My Jabber ID: ff119b40-9936-4b72-9c40-fa01ffa72a97@meet.jit.si/042b8499-0375-488e-9d55-b748684ee305 lib-jitsi-meet.min.js:2:299646
[modules/RTC/RTCUtils.js] <me</<.value/i<>:  Got media constraints:  Object { video: Object, audio: Object } lib-jitsi-meet.min.js:2:299646
[modules/xmpp/strophe.ping.js] <f</<.value>:  XMPP pings will be sent every 10000 ms lib-jitsi-meet.min.js:2:299646
[modules/RTC/RTCUtils.js] <E/<>:  Available devices:  Array [ MediaDeviceInfo, MediaDeviceInfo, MediaDeviceInfo ] lib-jitsi-meet.min.js:2:299646
[react/features/analytics/functions.js] <a/<>:  Loaded 2 analytics handlers app.bundle.min.js:2:725010
[modules/RTC/RTCUtils.js] <me</<.value/</<>:  onUserMediaSuccess lib-jitsi-meet.min.js:2:299646
[react/features/base/tracks/functions.js] <r/<>:  Failed to create local tracks Array [ "audio", "video" ] TypeError: i.getSettings is not a function
Stack trace:
s/<@https://cdn.jitsi.net/2441/libs/lib-jitsi-meet.min.js?v=2441:2:186863
s@https://cdn.jitsi.net/2441/libs/lib-jitsi-meet.min.js?v=2441:2:186719
A</<.value/<@https://cdn.jitsi.net/2441/libs/lib-jitsi-meet.min.js?v=2441:2:193118
T/</<@https://cdn.jitsi.net/2441/libs/app.bundle.min.js?v=2441:2:2369147
T/<@https://cdn.jitsi.net/2441/libs/app.bundle.min.js?v=2441:2:2369017
e.exports/l@https://cdn.jitsi.net/2441/libs/app.bundle.min.js?v=2441:2:1358193
 app.bundle.min.js:2:725010

[modules/RTC/RTCUtils.js] <me</<.value>:  Using the new gUM flow lib-jitsi-meet.min.js:2:299646
[modules/RTC/RTCUtils.js] <me</<.value/i<>:  Got media constraints:  Object { video: false, audio: Object } lib-jitsi-meet.min.js:2:299646
[modules/RTC/RTCUtils.js] <me</<.value/</<>:  onUserMediaSuccess lib-jitsi-meet.min.js:2:299646
[react/features/base/tracks/functions.js] <r/<>:  Failed to create local tracks Array [ "audio" ] TypeError: i.getSettings is not a function
Stack trace:
s/<@https://cdn.jitsi.net/2441/libs/lib-jitsi-meet.min.js?v=2441:2:186863
s@https://cdn.jitsi.net/2441/libs/lib-jitsi-meet.min.js?v=2441:2:186719
A</<.value/<@https://cdn.jitsi.net/2441/libs/lib-jitsi-meet.min.js?v=2441:2:193118
T/</<@https://cdn.jitsi.net/2441/libs/app.bundle.min.js?v=2441:2:2369147
T/<@https://cdn.jitsi.net/2441/libs/app.bundle.min.js?v=2441:2:2369017
e.exports/l@https://cdn.jitsi.net/2441/libs/app.bundle.min.js?v=2441:2:1358193
 app.bundle.min.js:2:725010

[modules/RTC/RTCUtils.js] <me</<.value>:  Using the new gUM flow lib-jitsi-meet.min.js:2:299646
[modules/RTC/RTCUtils.js] <me</<.value/i<>:  Got media constraints:  Object { video: Object, audio: false } lib-jitsi-meet.min.js:2:299646
[modules/RTC/RTCUtils.js] <me</<.value/</<>:  onUserMediaSuccess lib-jitsi-meet.min.js:2:299646
[react/features/base/tracks/functions.js] <r/<>:  Failed to create local tracks Array [ "video" ] TypeError: i.getSettings is not a function
Stack trace:
s/<@https://cdn.jitsi.net/2441/libs/lib-jitsi-meet.min.js?v=2441:2:186863
s@https://cdn.jitsi.net/2441/libs/lib-jitsi-meet.min.js?v=2441:2:186719
A</<.value/<@https://cdn.jitsi.net/2441/libs/lib-jitsi-meet.min.js?v=2441:2:193118
T/</<@https://cdn.jitsi.net/2441/libs/app.bundle.min.js?v=2441:2:2369147
T/<@https://cdn.jitsi.net/2441/libs/app.bundle.min.js?v=2441:2:2369017
e.exports/l@https://cdn.jitsi.net/2441/libs/app.bundle.min.js?v=2441:2:1358193
 app.bundle.min.js:2:725010

[conference.js] <t.a.init/<>:  initialized with %s local tracks 0 app.bundle.min.js:2:725010
[modules/xmpp/ChatRoom.js] <t>:  Joined MUC as probnasoba@conference.meet.jit.si/ff119b40 lib-jitsi-meet.min.js:2:299646
[modules/connectivity/ParticipantConnectionStatus.js] <e>:  RtcMuteTimeout set to: 2000 lib-jitsi-meet.min.js:2:299646
[modules/statistics/AvgRTPStatsReporter.js] <e>:  Avg RTP stats will be calculated every 15 samples lib-jitsi-meet.min.js:2:299646
[JitsiConference.js] <r>:  backToP2PDelay: 5 lib-jitsi-meet.min.js:2:299646
[modules/remotecontrol/RemoteControl.js] <g</<.value>:  Initializing remote control. app.bundle.min.js:2:725010
[conference.js] <t.a.updateAudioIconEnabled>:  Microphone button enabled: false local audio: null audio devices: undefined device count: 0 app.bundle.min.js:2:725010
[conference.js] <t.a.updateVideoIconEnabled>:  Camera button enabled: false local video: null video devices: undefined device count: 0 app.bundle.min.js:2:725010
[modules/xmpp/moderator.js] <i.prototype.setFocusUserJid>:  Focus jid set to:  undefined lib-jitsi-meet.min.js:2:299646
[modules/xmpp/moderator.js] <i.prototype.createConferenceIq>:  Session ID: null machine UID: a6cd0ce50c6c84e0fa733c34a0e12ca2 lib-jitsi-meet.min.js:2:299646
[modules/xmpp/moderator.js] <i.prototype.setFocusUserJid>:  Focus jid set to:  focus@auth.meet.jit.si lib-jitsi-meet.min.js:2:299646
[modules/xmpp/moderator.js] <i.prototype.parseConfigOptions>:  Authentication enabled: false lib-jitsi-meet.min.js:2:299646
[modules/xmpp/moderator.js] <i.prototype.parseConfigOptions>:  External authentication enabled: false lib-jitsi-meet.min.js:2:299646
[modules/xmpp/moderator.js] <i.prototype.parseConfigOptions>:  Sip gateway enabled:  true lib-jitsi-meet.min.js:2:299646
[conference.js] <t.a.updateAudioIconEnabled>:  Microphone button enabled: true local audio: null audio devices: [object MediaDeviceInfo],[object MediaDeviceInfo] device count: 2 app.bundle.min.js:2:725010
[conference.js] <t.a.updateVideoIconEnabled>:  Camera button enabled: true local video: null video devices: [object MediaDeviceInfo] device count: 1 app.bundle.min.js:2:725010
[modules/xmpp/ChatRoom.js] <w</<.value>:  entered probnasoba@conference.meet.jit.si/cf3a4d27 Object { show: "", affiliation: "owner", role: "moderator", jid: "cf3a4d27-aca7-4a74-bd70-881d82c9ebd…", isFocus: false, isHiddenDomain: false, nick: "Goran" } lib-jitsi-meet.min.js:2:299646
[conference.js] <t.a._setupListeners/<>:  USER %s connnected cf3a4d27 Object { _jid: "probnasoba@conference.meet.jit…", _id: "cf3a4d27", _conference: Object, _displayName: "Goran", _supportsDTMF: false, _tracks: Array[1], _role: "moderator", _status: null, _availableDevices: Object, _hidden: false, 3 more… } app.bundle.min.js:2:725010
[modules/UI/videolayout/RemoteVideo.js] <r.prototype.updateConnectionStatusIndicator>:  cf3a4d27 thumbnail connection status: active app.bundle.min.js:2:725010
[JitsiConference.js] <r.prototype._maybeStartOrStopP2P>:  P2P? isModerator: false, peerCount: 1 => true lib-jitsi-meet.min.js:2:299646
[modules/xmpp/ChatRoom.js] <w</<.value>:  entered probnasoba@conference.meet.jit.si/focus Object { show: "", affiliation: "owner", role: "moderator", jid: "focus@auth.meet.jit.si/focus1884853…", isFocus: true, isHiddenDomain: false } lib-jitsi-meet.min.js:2:299646
[modules/xmpp/ChatRoom.js] <w</<.value>:  Ignore focus: probnasoba@conference.meet.jit.si/focus, real JID: focus@auth.meet.jit.si/focus188485351444 lib-jitsi-meet.min.js:2:299646
[modules/UI/UI.js] <O.initEtherpad>:  Etherpad is enabled app.bundle.min.js:2:725010
[modules/version/ComponentsVersions.js] <r.prototype.processPresence/<>:  Got xmpp version: Prosody(trunk nightly build 747 (2017-02-19, 428d4abee723),Linux) lib-jitsi-meet.min.js:2:299646
[modules/version/ComponentsVersions.js] <r.prototype.processPresence/<>:  Got focus version: JiCoFo(1.0.1.0-388,Linux) lib-jitsi-meet.min.js:2:299646
[modules/version/ComponentsVersions.js] <r.prototype.processPresence/<>:  Got videobridge version: JVB(0.1.1011,Linux) lib-jitsi-meet.min.js:2:299646
[modules/xmpp/recording.js] <i.prototype.handleJibriPresence>:  Handle Jibri presence : off null lib-jitsi-meet.min.js:2:299646
[conference.js] <t.a._setupListeners/<>:  Received recorder status change:  available undefined app.bundle.min.js:2:725010
[conference.js] <t.a._setupListeners/<>:  My role changed, new role: moderator app.bundle.min.js:2:725010
[JitsiConference.js] <r.prototype._maybeStartOrStopP2P>:  P2P? isModerator: true, peerCount: 1 => true lib-jitsi-meet.min.js:2:299646
[JitsiConference.js] <r.prototype._maybeStartOrStopP2P>:  Everyone's a moderator - the other peer should start P2P ff119b40 cf3a4d27 lib-jitsi-meet.min.js:2:299646
[modules/xmpp/ChatRoom.js] <w</<.value>:  (TIME) MUC joined:	 3126.6800000000003 lib-jitsi-meet.min.js:2:299646
[modules/xmpp/ChatRoom.js] <w</<.value>:  chat 6455b5b5 xx lib-jitsi-meet.min.js:2:299646
[modules/xmpp/ChatRoom.js] <w</<.value>:  chat 6455b5b5 xx lib-jitsi-meet.min.js:2:299646
[modules/xmpp/ChatRoom.js] <w</<.value>:  chat 6455b5b5 xx lib-jitsi-meet.min.js:2:299646
[modules/xmpp/ChatRoom.js] <w</<.value>:  chat cf3a4d27 xx lib-jitsi-meet.min.js:2:299646
[modules/xmpp/ChatRoom.js] <w</<.value>:  chat cf3a4d27 xx lib-jitsi-meet.min.js:2:299646
[modules/xmpp/ChatRoom.js] <w</<.value>:  chat cf3a4d27 xx lib-jitsi-meet.min.js:2:299646
[modules/xmpp/ChatRoom.js] <w</<.value>:  chat cf3a4d27 xx lib-jitsi-meet.min.js:2:299646
[modules/xmpp/ChatRoom.js] <w</<.value>:  chat 6455b5b5 xx lib-jitsi-meet.min.js:2:299646
[modules/xmpp/ChatRoom.js] <w</<.value>:  chat cf3a4d27 xx lib-jitsi-meet.min.js:2:299646
[modules/xmpp/ChatRoom.js] <w</<.value>:  chat 6455b5b5 xx lib-jitsi-meet.min.js:2:299646
[modules/xmpp/ChatRoom.js] <w</<.value>:  chat 6455b5b5 xx lib-jitsi-meet.min.js:2:299646
[modules/xmpp/ChatRoom.js] <w</<.value>:  chat cf3a4d27 xx lib-jitsi-meet.min.js:2:299646
[modules/xmpp/ChatRoom.js] <w</<.value>:  chat 6455b5b5 xx lib-jitsi-meet.min.js:2:299646
[modules/xmpp/ChatRoom.js] <w</<.value>:  chat 6455b5b5 xx lib-jitsi-meet.min.js:2:299646
[modules/xmpp/ChatRoom.js] <w</<.value>:  chat cf3a4d27 xx lib-jitsi-meet.min.js:2:299646
[modules/xmpp/ChatRoom.js] <w</<.value>:  Subject is changed to  lib-jitsi-meet.min.js:2:299646
[modules/UI/videolayout/LargeVideoManager.js] <x</<.value/<>:  hover in %s ff119b40 app.bundle.min.js:2:725010
[modules/xmpp/strophe.jingle.js] <_</<.value>:  on jingle session-initiate from probnasoba@conference.meet.jit.si/focus <iq type="set" to="ff119b40-9936-4b72-9c40-fa01ffa72a97@meet.jit.si/042b8499-0375-488e-9d55-b748684ee305" from="probnasoba@conference.meet.jit.si/focus" id="ZmYxMTliNDAtOTkzNi00YjcyLTljNDAtZmEwMWZmYTcyYTk3QG1lZXQuaml0LnNpLzA0MmI4NDk5LTAzNzUtNDg4ZS05ZDU1LWI3NDg2ODRlZTMwNQBvOU05UC04NjMzOAC6Rj15tQ6tVzVzCxZhcKoR" xmlns="jabber:client"> lib-jitsi-meet.min.js:2:299646
[modules/xmpp/strophe.jingle.js] <_</<.value>:  (TIME) received session-initiate:	 3851.81 lib-jitsi-meet.min.js:2:299646
[modules/xmpp/strophe.jingle.js] <_</<.value>:  Marking session from probnasoba@conference.meet.jit.si/focus as *not* P2P lib-jitsi-meet.min.js:2:299646
SdpSimulcast: using 3 layers lib-jitsi-meet.min.js:2:896336
[modules/RTC/TraceablePeerConnection.js] <r>:  Create new TPC[1,p2p:false] lib-jitsi-meet.min.js:2:299646
[modules/RTC/BridgeChannel.js] <e>:  constructor() with peerconnection lib-jitsi-meet.min.js:2:299646
[JitsiConference.js] <r.prototype.onIncomingCall>:  Starting CallStats for JVB connection... lib-jitsi-meet.min.js:2:299646
[modules/statistics/statistics.js] <a.prototype.startCallStats>:  Starting CallStats for TPC[1,p2p:false]... lib-jitsi-meet.min.js:2:299646
[modules/RTC/TraceablePeerConnection.js] <O.localDescription>:  getLocalDescription no localDescription found lib-jitsi-meet.min.js:2:299646
[modules/xmpp/JingleSessionPC.js] <E</<.value>:  Renegotiate: setting remote description lib-jitsi-meet.min.js:2:299646
Halt: There are no SSRC groups in the remote description. lib-jitsi-meet.min.js:2:897276
[modules/xmpp/RtxModifier.js] <l</<.value>:  RtxModifier doing nothing, no video ssrcGroups present lib-jitsi-meet.min.js:2:299646
[modules/xmpp/JingleSessionPC.js] <E</<.value/<>:  Renegotiate: creating answer lib-jitsi-meet.min.js:2:299646
[modules/RTC/TraceablePeerConnection.js] <r.prototype._createOfferOrAnswer/s>:  Got local SSRCs MAP:  Map {  } lib-jitsi-meet.min.js:2:299646
[modules/xmpp/JingleSessionPC.js] <E</<.value/</<>:  Renegotiate: setting local description lib-jitsi-meet.min.js:2:299646
[modules/xmpp/JingleSessionPC.js] <E</<.value>:  Sending session-accept <iq to="probnasoba@conference.meet.jit.si/focus" type="set" xmlns="jabber:client" id="d42bde5b-1eab-4927-967d-013a94e992af:sendIQ"> lib-jitsi-meet.min.js:2:299646
[modules/RTC/TraceablePeerConnection.js] <r.prototype._remoteTrackAdded>:  TPC[1,p2p:false] remote track added: e14e700d-f767-4cf5-9b1f-18183d31b5f2 audio lib-jitsi-meet.min.js:2:299646
[modules/RTC/TraceablePeerConnection.js] <r.prototype._remoteTrackAdded>:  TPC[1,p2p:false] associated ssrc cf3a4d27 2129926492 lib-jitsi-meet.min.js:2:299646
[modules/RTC/TraceablePeerConnection.js] <r.prototype._remoteStreamAdded>:  TPC[1,p2p:false] ignored remote 'stream added' event for non-user streamid: mixedmslabel lib-jitsi-meet.min.js:2:299646
[modules/xmpp/JingleSessionPC.js] <E</<.value/this.peerconnection.oniceconnectionstatechange>:  (TIME) ICE checking P2P? false:	 4415.445 lib-jitsi-meet.min.js:2:299646
[modules/statistics/CallStats.js] <f</<.value>:  CallStats Status: err=success msg=SDK authentication successful. lib-jitsi-meet.min.js:2:299646
[modules/statistics/CallStats.js] <f</<.value>:  addNewFabric jitsi lib-jitsi-meet.min.js:2:299646
reportError RTCPeerConnection { _onicecandidate: r/this.peerconnection.onicecandidate(), _eventMap: Object, localDescription: RTCSessionDescription, remoteDescription: RTCSessionDescription, signalingState: "stable", iceGatheringState: "complete", iceConnectionState: "connected", peerIdentity: Promise, idpLoginUrl: null, onnegotiationneeded: r/this.peerconnection.onnegotiationneeded() } meet.jit.si/probnasoba applicationLog lib-jitsi-meet.min.js:2:439253
reportError RTCPeerConnection { _onicecandidate: r/this.peerconnection.onicecandidate(), _eventMap: Object, localDescription: RTCSessionDescription, remoteDescription: RTCSessionDescription, signalingState: "stable", iceGatheringState: "complete", iceConnectionState: "connected", peerIdentity: Promise, idpLoginUrl: null, onnegotiationneeded: r/this.peerconnection.onnegotiationneeded() } meet.jit.si/probnasoba applicationLog lib-jitsi-meet.min.js:2:439253
reportError RTCPeerConnection { _onicecandidate: r/this.peerconnection.onicecandidate(), _eventMap: Object, localDescription: RTCSessionDescription, remoteDescription: RTCSessionDescription, signalingState: "stable", iceGatheringState: "complete", iceConnectionState: "connected", peerIdentity: Promise, idpLoginUrl: null, onnegotiationneeded: r/this.peerconnection.onnegotiationneeded() } meet.jit.si/probnasoba applicationLog lib-jitsi-meet.min.js:2:439253
reportError RTCPeerConnection { _onicecandidate: r/this.peerconnection.onicecandidate(), _eventMap: Object, localDescription: RTCSessionDescription, remoteDescription: RTCSessionDescription, signalingState: "stable", iceGatheringState: "complete", iceConnectionState: "connected", peerIdentity: Promise, idpLoginUrl: null, onnegotiationneeded: r/this.peerconnection.onnegotiationneeded() } meet.jit.si/probnasoba applicationLog lib-jitsi-meet.min.js:2:439253
reportError RTCPeerConnection { _onicecandidate: r/this.peerconnection.onicecandidate(), _eventMap: Object, localDescription: RTCSessionDescription, remoteDescription: RTCSessionDescription, signalingState: "stable", iceGatheringState: "complete", iceConnectionState: "connected", peerIdentity: Promise, idpLoginUrl: null, onnegotiationneeded: r/this.peerconnection.onnegotiationneeded() } meet.jit.si/probnasoba applicationLog lib-jitsi-meet.min.js:2:439253
reportError RTCPeerConnection { _onicecandidate: r/this.peerconnection.onicecandidate(), _eventMap: Object, localDescription: RTCSessionDescription, remoteDescription: RTCSessionDescription, signalingState: "stable", iceGatheringState: "complete", iceConnectionState: "connected", peerIdentity: Promise, idpLoginUrl: null, onnegotiationneeded: r/this.peerconnection.onnegotiationneeded() } meet.jit.si/probnasoba applicationLog lib-jitsi-meet.min.js:2:439253
[modules/statistics/CallStats.js] <f</<.value>:  CallStats Status: err=success msg=WebSocket establishment successful. lib-jitsi-meet.min.js:2:299646
[modules/xmpp/JingleSessionPC.js] <E</<.value>:  sendIceCandidate: last candidate. lib-jitsi-meet.min.js:2:299646
(TIME) Render audio:	 4488.165 lib-jitsi-meet.min.js:2:878544
(TIME) TTFM audio:	 3724.3650000000002 lib-jitsi-meet.min.js:2:878935
[modules/xmpp/JingleSessionPC.js] <E</<.value>:  sendIceCandidates Array [ RTCIceCandidate ] lib-jitsi-meet.min.js:2:299646
[modules/xmpp/JingleSessionPC.js] <E</<.value/this.peerconnection.oniceconnectionstatechange>:  (TIME) ICE connected P2P? false:	 4773.735 lib-jitsi-meet.min.js:2:299646
[modules/UI/videolayout/LargeVideoManager.js] <x</<.value/<>:  hover in %s cf3a4d27 app.bundle.min.js:2:725010
[modules/UI/videolayout/RemoteVideo.js] <r.prototype.updateConnectionStatusIndicator>:  cf3a4d27 thumbnail connection status: active app.bundle.min.js:2:725010
reportError RTCPeerConnection { _onicecandidate: r/this.peerconnection.onicecandidate(), _eventMap: Object, localDescription: RTCSessionDescription, remoteDescription: RTCSessionDescription, signalingState: "stable", iceGatheringState: "complete", iceConnectionState: "connected", peerIdentity: Promise, idpLoginUrl: null, onnegotiationneeded: r/this.peerconnection.onnegotiationneeded() } meet.jit.si/probnasoba applicationLog lib-jitsi-meet.min.js:2:439253

```

**Comment by kwill** [comment]
This issue is resolved in PureBrowser 52.5.0
